src/components/material/MaterialPeriodical.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import { FC } from "react";
+import { useGetHoldingsV3 } from "../../core/fbs/fbs";
+import { groupObjectArrayByProperty } from "../../core/utils/helpers/general";
+import { FaustId } from "../../core/utils/types/ids";
+import MaterialPeriodicalSelect from "./MaterialPeriodicalSelect";
+
+export interface MaterialPeriodicalProps {
+  faustId: FaustId;
+  selectPeriodikumSelect: (periodikumSelect: string | null) => void;
+}
+
+const MaterialPeriodical: FC<MaterialPeriodicalProps> = ({
+  faustId,
+  selectPeriodikumSelect
+}) => {
+  const { data, isLoading, isError } = useGetHoldingsV3({
+    recordid: [String(faustId)]
+  });
+
+  if (isLoading || isError || !data) return null;
+
+  const holdings = data[0]?.holdings;
+  if (!holdings) {
+    return null;
```

Comment:
Please remove null chaining and checks.

As far as I can tell they should not be necessary.